kubernetes_platform/python/requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+docformatter==1.4
```

Comment:
Optional: It'd be nice if this also included the `requirements.txt` to do one pip install.